### ISSUE TITLE
relative hrefs in included layouts

### ISSUE THREAD

**Issue by sdotz** [gjtorikian/graphql-docs#13]
I had some issues with links going to relative places with the up-one-levels 

`<a href="../../object/mutation" class="sidebar-link<% if title == "Mutation" %> current<% end %>">`

Are the docs intended to be embedded on a site, or served as a standalone site?

Either way I think it would be useful to express the paths as relative to the site root and optionally prefixed with some user configurable path

`<user_prefix>/object/objectname`

What do you think?

**Comment by swalkinshaw** [comment]
I had the same need so I created a helper like this:

```ruby
def link(name: nil, kind: nil, base: @options[:pipeline_config][:context][:base_url])
  [base, kind, slugify(name)].join('/')
end
```

**Comment by gjtorikian** [comment]
This should now be fixed in 0.2.0! Sorry for the delay in response. The option is called `base_url`.
